src/PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.cpp

```diff
@@ -37,18 +37,123 @@ tnsr::aa<DataType, Dim, Fr> compute_spacetime_metric(
   return spacetime_metric;
 }
 
+template <size_t Dim, typename Fr, typename DataType>
+tnsr::AA<DataType, Dim, Fr> compute_inverse_spacetime_metric(
+    const Scalar<DataType>& lapse, const tnsr::I<DataType, Dim, Fr>& shift,
+    const tnsr::II<DataType, Dim, Fr>& inverse_spatial_metric) noexcept {
+  tnsr::AA<DataType, Dim, Fr> inverse_spacetime_metric{};
+
+  inverse_spacetime_metric.template get<0, 0>() =
+      -1.0 / (lapse.get() * lapse.get());
```

Comment:
Since the quantity `1/lapse^2` is used many times in this function, perhaps it can be precomputed?

(Or to be clever with storage, `-inverse_spacetime_metric.template get<0,0>` could replace `1/lapse^2` in the code below, but this reduces clarity...)

Reply:
You could do it with a reference: `const auto& minus_one_over_lapse_squared = inverse_spacetime_metric.template get<0,0>()` or something.